### PR TITLE
feat home-page

### PR DESCRIPTION
**Pull request by iamsaurav28**

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
Looks good to me. No changes suggested.

**Review by shubhamkumar648 — APPROVED** [review]
Everything looks good ,good folder structure

File: quiz--project/src/home/Question.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+  return (
+    <>
+    
+    </>
+  );
+};
+
+export default Question;
```
Comment (abhij1607) [inline quiz--project/src/home/Question.js:14]: Hope you will modify this in future, empty component rn

File: quiz--project/src/styles.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
+* {
+     margin: 0;
+     padding: 0;
+     /* box-sizing: border-box; */
```
Comment (abhij1607) [inline quiz--project/src/styles.css]: remove comments

File: quiz--project/src/home/Question.js (hunk 1 above)
Comment (iamsaurav28) [reply quiz--project/src/home/Question.js:14]: yes i will

File: quiz--project/src/styles.css (hunk 2 above)
Comment (iamsaurav28) [reply quiz--project/src/styles.css]: ok

File: quiz--project/src/styles.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+     width: 100%;
+     transition: ease 0.1s;
+   }
+   .inner-wrap button:hover {
+     background: #161d24;
+     color: rgb(223, 223, 223);
+   }
+   .error {
+     text-align: center;
+     font-size: 1rem !important;
```
Comment (abhij1607) [inline quiz--project/src/styles.css:72]: avoid use of important

Comment (iamsaurav28) [reply quiz--project/src/styles.css:72]: okk

File: quiz--project/src/styles.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+     /* box-sizing: border-box; */
+   }
+   body {
+     font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+       "Lucida Sans", Arial, sans-serif;
+   }
+   /* H O M E */
+   .wrapper,
+   .question-wrap {
+     background: #4b5c69;
```
Comment (abhij1607) [inline quiz--project/src/styles.css:13]: use variables

**Review by abhij1607 — COMMENTED** [review]
Looks good to merge

Suggestions:
define variables for color and other units
avoid use of !important, try to use rule of specificity
provide some description in PR

File: quiz--project/src/styles.css (hunk 4 above)
Comment (iamsaurav28) [reply quiz--project/src/styles.css:13]: okk

**Pull request merged**